polkadot/node/network/collator-protocol/src/collator_side/mod.rs

```diff
@@ -464,15 +476,15 @@ async fn distribute_collation<Context>(
 		state.collation_result_senders.insert(candidate_hash, result_sender);
 	}
 
+	let parent_head_data = if elastic_scaling {
```

Comment:
I think it would be better to add hide this behind cargo feature so we can merge the PR right now to unblock testing as soon as possible.